### PR TITLE
SpringBoot 3.0: Add noship features

### PR DESCRIPTION
**Pull request by brenthdaniel**

### CODE REVIEW

**Comment by brenthdaniel** [comment]
#build

**Comment by brenthdaniel** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/springBoot-3.0/resources/l10n/com.ibm.websphere.appserver.springBoot-3.0.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables support for Spring Boot applications using Spring Boot \
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/springBoot-3.0/resources/l10n/com.ibm.websphere.appserver.springBoot-3.0.properties]:
```suggestion
description=This feature enables support for Spring Boot applications that use Spring Boot \
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review- minor edit per IBM style

**Comment by brenthdaniel** [comment]
#build

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Comment by posmith** [comment]
L2 message review not required

**Comment by hlhoots** [comment]
Closing this PR, as it was picked up in the following PR with additional updates: https://github.com/OpenLiberty/open-liberty/pull/25329

**Pull request closed without merging**